sonic-sensormond/scripts/sensormond

```diff
@@ -0,0 +1,561 @@
+#!/usr/bin/python3
+
+"""
+    sensormond
+    Sensor monitor daemon for SONiC
+"""
+
+import signal
+import sys
+import threading
+import time
+from datetime import datetime
```

Comment:
We shouldn't need both `time` and `datetime`, can we stick to just one time module?

Reply:
k